connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -1990,14 +2019,14 @@ class Dataset[T] private[sql] (val sparkSession: SparkSession, private[sql] val
       viewName: String,
       replace: Boolean,
       global: Boolean): Unit = {
-    val command = session.newCommand { builder =>
+    val command = sparkSession.newCommand { builder =>
```

Comment:
Ah yes, I guess I broke the build...

Reply:
https://github.com/apache/spark/pull/40174

Reply:
Or we use this PR to fix the build, depending on you.